src/Symfony/Component/DependencyInjection/Config/AutowireServiceResource.php

```diff
@@ -33,20 +32,7 @@ public function isFresh($timestamp)
             return false;
         }
 
-        // has the file *not* been modified? Definitely fresh
-        if (@filemtime($this->filePath) <= $timestamp) {
-            return true;
-        }
-
-        try {
-            $reflectionClass = new \ReflectionClass($this->class);
-        } catch (\ReflectionException $e) {
-            // the class does not exist anymore!
-
-            return false;
-        }
-
-        return AutowirePass::createResourceForClass($reflectionClass);
+        return @filemtime($this->filePath) <= $timestamp;
```

Comment:
@weaverryan Yes, I tried to figure out what to do, I was not sure about that "easy" way :)

Thanks for the lead, I'm on it!